tensorflow_addons/image/transform_ops.py

```diff
@@ -89,12 +89,14 @@ def transform(images,
         if len(transform_or_transforms.get_shape()) == 1:
             transforms = transform_or_transforms[None]
         elif transform_or_transforms.get_shape().ndims is None:
-            raise TypeError(
+            raise ValueError(
                 "transform_or_transforms rank must be statically known")
```

Comment:
maybe make the variable name same with input argument
```suggestion
            raise ValueError(
                "transforms rank must be statically known")
```